packages/server/src/threads/automation.js

```diff
@@ -264,14 +261,11 @@ class Orchestrator {
         inputs = automationUtils.cleanInputValues(inputs, step.schema.inputs)
         try {
           // appId is always passed
-          let tenantId = app.tenantId || DEFAULT_TENANT_ID
-          const outputs = await doInTenant(tenantId, () => {
-            return stepFn({
-              inputs: inputs,
-              appId: this._appId,
-              emitter: this._emitter,
-              context: this._context,
-            })
+          const outputs = await stepFn({
```

Comment:
It's safe to remove the `doInTenant` as we're already doing a `doInAppContext` futher up the chain?